geth/api/backend_test.go

```diff
@@ -256,6 +258,54 @@ func (s *BackendTestSuite) TestCallRPC() {
 	}
 }
 
+func (s *BackendTestSuite) TestCallRPCSendTransaction() {
```

Comment:
Can this test pass at least once?

Reply:
No idea. Just copied this test from other place.

Reply:
I can get it passed in Ropsten easily but it does want to in RinkebyNetworkID :/

Reply:
Let's change it to Ropsten.